src/gui/about.cpp

```diff
@@ -27,7 +27,7 @@
 #include "ui_about.h"
 #include "../jacktrip_globals.h"
 
-const QString About::sBuildID = "2021052500";
+const QString About::sBuildID = "2021052600";
```

Comment:
I think it would be good to talk about version identifiers in general - see https://github.com/jacktrip/jacktrip/issues/284

Why do we need a build ID?

Reply:
I was using the build ID (and still do) to track my binary releases because QJackTrip dev was happening at a different pace to JackTrip dev. By default this is probably not needed any more, and should probably be set at build time, not in the code. Will update.